src/components/video.js

```diff
@@ -154,23 +126,28 @@ const Video = () => {
               })
 
               participant.on("trackSubscribed", (track) => {
-                setLocal(false)
-                setRemote(true)
                 document.getElementById("remote-media-div").appendChild(track.attach())
               })
             })
           },
           (error) => {
             console.error(`Unable to connect to Room: ${error.message}`)
           }))
+      .catch((err) => {
+        console.log("There was an error", err)
+      })
   }
 
+  useEffect(() => {
+    if (state.user) {
+      call()
+    }
+  }, [state.user])
```

Comment:
Codacy found an issue: [React Hook useEffect has a missing dependency: 'call'. Either include it or remove the dependency array.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)